packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -148,6 +153,9 @@ function dispatchUserBlockingUpdate(
   container,
   nativeEvent,
 ) {
+  // TODO: Double wrapping is necessary while we decouple Scheduler priority.
+  const previousPriority = getCurrentUpdateLanePriority();
+  setCurrentUpdateLanePriority(InputContinuousLanePriority);
```

Comment:
Needs a try-catch, since runWithPriority could throw

Reply:
I see, I misunderstood "we should be able to mutate the value directly without a wrapper function" as not needing a try/catch.